CHANGELOG.md

```diff
@@ -17,19 +18,56 @@ respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the
 previous public API version
 
+# r1.2
+## Release Notes
+
+This release contains the definition and documentation of
+* connectivity-insights v0.4.0
+* connectivity-insights-subscriptions v0.4.0
+* application-profiles v0.3.0
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
+
+*  **r1.2 release of connectivity insights has the following API definitions:**
+
+    *  **connectivity-insights v0.4.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml)
+
+    *  **connectivity-insights-subscriptions v0.4.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
+
+    *  **application-profiles v0.3.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
+
+### Added
+* n/a
+### Changed
+* n/a
+
+**Full Changelog**: https://github.com/camaraproject/connectivityInsights/compare/r1.1...r1.2
```

Comment:
This line - if included - should reflect all changes since the last public release, not compared to last pre-release. In your case - compared to no previous release - it would be:

```suggestion
**Full Changelog**: https://github.com/camaraproject/connectivityInsights/commits/r1.2
```